components/kyma-environment-broker/internal/broker/broker.go

```diff
@@ -146,6 +132,19 @@ func (b *KymaEnvBroker) Provision(ctx context.Context, instanceID string, detail
 		SecretName: b.Config.SecretName,
 	}
 
+	err = b.Storage.Instances().Insert(internal.Instance{
+		InstanceID:             instanceID,
+		RuntimeID:              "",
+		GlobalAccountID:        ersContext.GlobalaccountID,
+		ServiceID:              details.ServiceID,
+		ServicePlanID:          details.PlanID,
+		DashboardURL:           fixedDummyURL,
+		ProvisioningParameters: string(details.RawParameters),
```

Comment:
let's discuss `subaccount_id` and SM parameters, should we save it also?

Reply:
discussed, we don't need them